myosuite/envs/myo/myochallenge/chasetag_v0.py

```diff
@@ -37,11 +37,13 @@ class ChallengeOpponent:
     Contains several different policies. For the final evaluation, an additional
     non-disclosed policy will be used.
     """
-    def __init__(self, sim, rng, probabilities: list, min_spawn_distance: float):
+    def __init__(self, sim, rng, probabilities: list, min_spawn_distance: float, linear_velocity: float):
         self.dt = 0.01
         self.sim = sim
         self.opponent_probabilities = probabilities
         self.min_spawn_distance = min_spawn_distance
+        self.randomize_vel = True if linear_velocity == 'random' else False
```

Comment:
linear_velocity is passed as float and then treated as string

Reply:
fixed